tests/integration/models/sec10k/extract_test.py

```diff
@@ -0,0 +1,38 @@
+"""Validate basic 10k and exhibit 21 extraction."""
+
+import logging
+
+import dotenv
+import pytest
+from mozilla_sec_eia.models import sec10k
+
+logger = logging.getLogger(f"catalystcoop.{__name__}")
+
+
+def test_basic_10k_validation(
+    test_tracker_factory,
+    get_most_recent_mlflow_run_factory,
+):
+    """Test basic_10k_validation_job."""
+    dotenv.load_dotenv()
+    sec10k.defs.get_job_def("basic_10k_extraction_validation").execute_in_process()
+
+    run = get_most_recent_mlflow_run_factory("basic_10k_extraction_validation")
+
+    assert run.data.metrics["precision"] == 1
+    assert run.data.metrics["recall"] == 1
+
+
+@pytest.mark.xfail
```

Comment:
`xfail` due to weird errors on github runner from `xhtml2pdf`. Passing locally for me. Maybe @katie-lamb can try to run this test locally too?

Reply:
Actually I think we might just be running out of memory on the runner. Gotta figure something out...

Reply:
The PDF generation only runs 1 doc at a time, but we still run out of memory? That seems unexpected... did you end up figuring anything out here?